addons/website/i18n/website.pot

```diff
@@ -10883,7 +10883,7 @@ msgstr ""
 #. module: website
 #: code:addons/website/models/website.py:0
 #, python-format
-msgid "You must keep at least one website."
+msgid "You cannot delete default website %s. Try to change its settings instead"
```

Comment:
Generally try to avoid manually changing a `.pot`, just regenerate it from the Odoo UI and add it where it is in the exported file, in this case it should be moved above (exported/sorted by alphabetical order).
It will be easier later when we export translation and compare with the `.pot` file of the repo, as it won't be considered as outdated diff (since wrongly positioned).